app/src/main/java/com/protify/protifyapp/features/events/AddEvent.kt

```diff
@@ -457,13 +460,11 @@ open class AddEvent {
                 }
                 item {
                     EventOutdoorsItem(
-                        isOutside = isOutside,
                         onCheckedChange = { isChecked -> updateIsOutside(isChecked) }
                     )
                 }
                 item {
                     EventAIItem(
-                        isOptimized = isOptimized,
```

Comment:
Removing because this causes an infinite recompose error. isOutside is already declared globally and doesn't need to be passed in here